packages/perseus/src/components/math-input.tsx

```diff
@@ -330,25 +332,38 @@ class InnerMathInput extends React.Component<InnerProps, State> {
                         opened={this.state.keypadOpen}
                         onClose={() => this.closeKeypad()}
                         dismissEnabled
+                        aria-label={this.context.strings.mathInputTitle}
+                        aria-describedby="popover-content"
                         content={() => (
-                            <PopoverContentCore
-                                closeButtonVisible
-                                style={styles.popoverContent}
-                            >
-                                <DesktopKeypad
-                                    onAnalyticsEvent={
-                                        this.props.analytics.onAnalyticsEvent
-                                    }
-                                    extraKeys={this.props.extraKeys}
-                                    onClickKey={this.handleKeypadPress}
-                                    cursorContext={this.state.cursorContext}
-                                    convertDotToTimes={
-                                        this.props.convertDotToTimes
-                                    }
-                                    {...(this.props.keypadButtonSets ??
-                                        mapButtonSets(this.props?.buttonSets))}
-                                />
-                            </PopoverContentCore>
+                            <>
+                                <HeadingMedium
+                                    id="popover-content"
```

Comment:
Right now it's possible to have multiple keypads open at the same time. Should we auto-generate an ID here to make sure everything connects correctly?

Reply:
Good call out! Updated to use the first 8 chars of a UUID - the new ids are `popover-content-${shortened-uuid}`